src/Raven.Server/Config/Categories/TombstoneConfiguration.cs

```diff
@@ -13,5 +13,17 @@ public class TombstoneConfiguration : ConfigurationCategory
         [ConfigurationEntry("Tombstones.CleanupIntervalInMin", ConfigurationEntryScope.ServerWideOrPerDatabase)]
         [Description("Time (in minutes) between tombstone cleanups.")]
         public TimeSetting CleanupInterval { get; set; }
+
+        [DefaultValue(14)]
+        [TimeUnit(TimeUnit.Days)]
```

Comment:
If we allow shorter time periods, should this be in days?
What is I want this to be 36 hours?